spec/models/tag_spec.rb

```diff
@@ -46,4 +46,13 @@
 
     it { expect(child_tag).not_to be_valid }
   end
+
+  context 'with an activity_tag' do
+    subject { create(:tag) }
+    let!(:activity_tag) { create(:activity_tag, tag: subject) }
+
+    describe 'destroying' do
+      it { expect { subject.destroy! }.to raise_error }
```

Comment:
```suggestion
      it { expect { subject.destroy! }.to raise ActiveRecord::RecordNotDestroyed }
```

To avoid rubocop warning: 
WARNING: Using the `raise_error` matcher without providing a specific error or message risks false positives, since `raise_error` will match when Ruby raises a `NoMethodError`, `NameError` or `ArgumentError`, potentially allowing the expectation to pass without even executing the method you are intending to call. Actual error raised was #<ActiveRecord::RecordNotDestroyed: Failed to destroy the record>. Instead consider providing a specific error class or message.